.travis.yml

```diff
@@ -35,6 +35,22 @@ after_success:
   - scripts/deploy.sh netlify
   - scripts/push-image.sh unlock
   - scripts/push-image.sh unlock-integration
+  - scripts/package_locksmith.sh
 notifications:
   slack:
     secure: q4BMSH3FifcaoPwQ2QFocbv8OuNT4JWgeJuE7jxppw86rCJHOyp6SdLpzp4944cWTlEGFdbQsp83OE2fx99i9OncvlJM5Is1IQgIDUfOD2l8mKx+HPlWvuBLaRB5j7vdamzODaHU2NlS+XSU7I0ytxp39x5arYY5hR9yqETvkyjAsSWBjY74Vv+eED+ALtQD9HKoH3xaxgV5rolujR3MEzdMSoNpenivz+BIfKhGQlOHyEN0FOsZtBkCmJTZ9mQbPApu2NKQZRU2cTn7h4Pp4Gpy6mVPWtesUd9OtblsA658qw9Sn/C9hIO1uKgGb4G1PJrKaa/qs2WeMj3VNQ0CkubXYNUS+OfnRb0CT+eTRCndnhFsDdt0zgIBtFeG2FT83OD0sT1SHduY69uETxxstTJw6F8+q/n4JNImBT7klBn2tLrS8kOpk2dqIkpQsvKst67UaYVNa4x2wOcbBxwE+RXP3i3Hs+NpNW/3W9IJOXA2Twx6Q70ORlg3Lnfw28gTDOnh3ZYx48b8y17U4J4KJgEu7aaQCUtoPK/dSwGl0atMQn/FRwEHvIs7zIh/PqyWYRA+y0V3VAe2bAuLf5WTjUijN5w+wfHDGjo6fxZ6dPwBXhhbeFWl4Tz6GEiOGJsebJTtDKWYD/uItNqb3uFiEwEeUEFJCY1zVU5jT8NL3TY=
+deploy:
+  provider: elasticbeanstalk
+  access_key_id: 
+    secure: "jwqleo0h2qdUQ9w3+BkLrVKuXHNcY9072XGYFjuxJNkYJxfCBODaanm90hNcNNal6fqV/fkb+ilcvad4DBfpbUdkVTx+YSId8zEL1W99D1G1rJ28vs0nl+RYfbYutQUCvAWWD0wpTLDmcPNOFUBJc/RIiEeeqAywOueHAV1K4srYW3IjJWYskb0EdW+hANmLAnNkKboiPWj7FH9HoizH+Owt3apBdAkJ+4efKyyRCIZmzGPF3ymLezHhiWWu57WQtM4yo+1ipvqCQkzaC+rmVf+1YZUXf5HXeuhmyp+A4vn8x6Oy3V+PVwazh1ro4ylxpRXKMd/ia59iQ3QNcUsPlfE3Ew9g98hKKW9/QMwRCUxenp9ThmDwXvzoy29oMI+7xag03IiOhjR8gaKYADlt8vuaCwTuKruuKNgnfeBKhJH+Batgji508SbvNZ+M18W/4W3gn3JLbBiabpcTO/zpTbKwfUnAfBpsPNWwoQ9GlwSLF9FYapaS61ycYvKXCNTX/fsvB59aH9cD/H9jO65L7E9Y/fZbR/K+HoFEDKj43ZKUg/H68hwDLMKgx+legv8v3J6T2/XLPOdAw2E4pdRlIMkWSotithO5LPSsV4JXHlw148G41Cs+NRvcfj2tbqTZgEy+7ZsgFT3FV2nhfK6lFlYSr6vJUX9pOHA01d7Rjc4="
```

Comment:
Is there a way we can avoid saving this in our public git?

Reply:
eg, can we save them in Travis environment variables?

Reply:
@benwerd it should be possible to setup the config via the travis cli, i'm not sure how opaque this will be on UI side, but i'll give it a go to see if we can eliminate this all together